composer.json

```diff
@@ -21,10 +21,10 @@
     ],
     "require": {
         "php": "^7.0",
-        "laravel/framework": "5.5.*|5.6.*|5.7.*"
+        "illuminate/support": "^5.5"
```

Comment:
^5.5 will also accept 5.8 and 5.9, and we're not sure this package will work there.

Please modify your PR to only change the required package.

Reply:
I was update my PR